packages/agent/src/utils/random.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generates a random unsigned 32-bit integer between 0 and 0xffffffff
+ * @returns {number} a random number
+ */
+export const randomNumber = () => {
```

Comment:
appropriate/useful?
```suggestion
export const randomNumber = (): number => {
```

Reply:
This is possibly the most controversial question in the TypeScript community 😁. I've chosen not to enforce return types for this codebase. 

Broadly speaking, this type is trivially inferred, so it does not add any utility for someone importing the function